src/Encryption.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\EncryptionApiClient;
+
+class Encryption extends Common
+{
+    public function __construct(string $sapiToken, array $config)
+    {
+        parent::__construct(['X-StorageApi-Token' => $sapiToken], $config);
+    }
+
+    public function encryptPlainTextForConfiguration(
+        string $value,
+        string $projectId,
+        string $componentId,
+        string $configId
+    ): string {
+        $queryParams = [
+            'projectId' => $projectId,
+            'componentId' => $componentId,
+            'configId' => $configId,
+        ];
+
+        $url = 'encrypt';
+        if (count($queryParams) > 0) {
+            $url .= '?' . http_build_query($queryParams);
+        }
+
+        $result = $this->apiPost($url, ['#value' => $value]);
+        return $result['#value'];
```

Comment:
Obchazim ze `Common`  umi jen json request/response :trollface:

Reply:
ouch.. jako taky asi budu mhourit :eye: Aspon bych sem hodil comment, ze je to workaround a bylo by dobry to vycistit, az se to bude predelavat na normalniho clienta z platform-libraries.